### DISCUSSION TITLE
GitHub action to create pull request on public repository

### DISCUSSION THREAD

**Discussion by TharindaDilshan** [community/community#26385] [Actions]
<p>Is it possible to create pull requests on public repositories that reside in the same organization using GitHub actions?</p>

**Comment by kingthorin** [comment, marked as answer]
<p>Absolutely. Anything you can do in bash you can do in an action.</p>
<p>Here are some examples:</p>
<ul>
<li><a href="https://github.com/OWASP/wstg/blob/cb1adee9e9aac2eeba65a0971c7c8dec01efd4bf/.github/workflows/www_latest_update.yml" rel="noopener nofollow ugc">https://github.com/OWASP/wstg/blob/cb1adee9e9aac2eeba65a0971c7c8dec01efd4bf/.github/workflows/www_latest_update.yml</a></li>
<li><a href="https://github.com/OWASP/wstg/blob/cb1adee9e9aac2eeba65a0971c7c8dec01efd4bf/.github/workflows/build-json.yml" rel="noopener nofollow ugc">https://github.com/OWASP/wstg/blob/cb1adee9e9aac2eeba65a0971c7c8dec01efd4bf/.github/workflows/build-json.yml</a></li>
<li><a href="https://github.com/zaproxy/zap-extensions/blob/f153e78e7d1a1231bdb5ea4a7f761a48461e22c8/.github/workflows/create_wappalyzer_update.yml" rel="noopener nofollow ugc">https://github.com/zaproxy/zap-extensions/blob/f153e78e7d1a1231bdb5ea4a7f761a48461e22c8/.github/workflows/create_wappalyzer_update.yml</a></li>
<li><a href="https://github.com/OWASP/OWASP-VWAD/blob/1845377e47b4d1ac6f48e6e5456b21234d011bc0/.github/workflows/create_www_update.yml" rel="noopener nofollow ugc">https://github.com/OWASP/OWASP-VWAD/blob/1845377e47b4d1ac6f48e6e5456b21234d011bc0/.github/workflows/create_www_update.yml</a></li>
</ul>

**Comment by TharindaDilshan** [comment]
<p>Thank you for the references.</p>

**Comment by kingthorin** [comment]
<p>No problem, let me know if you have any questions. The secrets used for the push/PR are just PATs set in the Secrets of the repo or org running the workflow.</p>
